---
layout: default
title: research
---

<article class="row post-content">

<p>
<h2>
<strong>Research overview</strong></h2>
</p>
<h3>
<p>
My research group embraces the urgent need for <b>mammal phylogenetic ecology</b>, which I frame as the application of phylogeny-informed taxonomic knowledge to the study of species ecological interactions. More colloquially: <i>Know your species and their history to ask how and why organisms interact.</i>
</p>
<p>
Taxonomy is generally viewed as external to ecology, a fixed framework that is known without error rather than a variable—the taxonomic variable—that needs tuning to understand its impact on inferences. <b>Fossils</b> play a key role in this conception of phylogenetic ecology because they often represent extinct lineages, the absence of which in extant-only phylogenies tends to bias inferences of trait or lineage evolution. 
</p>
<p>
The dual importance of fossil and molecular insights is increasingly recognized; extant-only phylogenies cannot tell us everything we want to know about the past, but they can tell us about processes near the present day (often moreso than fossils). My recent work on the mammal tree of life has helped demonstrate this, showing that while fossil rate signatures are absent from older lineages of the extant tree, lineages younger than 10-million years (Ma) are unbiased (<a href="https://doi.org/10.1371/journal.pbio.3000494"><i>PLoS Biology</i> 2019</a>, <a href="https://doi.org/10.1016/j.cub.2021.07.012"><i>Current Biology</i> 2021</a>). Thus, integrating detailed molecular and ecological data from extant taxa relative to fossils and their preserved traits (e.g., proxies of body size, feeding mode, or dispersal ability) is the primary strategy of our lab for investigating eco-evolutionary questions.
</p>
<tr><td><img src="https://n8upham.github.io/images/Fig1new_jobs2021_overview_krat_text.jpg" width="1000px" style="padding-bottom: 20px; padding-top: 20px;"/></td></tr>
<br>
<p>
The groups is currently focusing on 3 main areas of research:
</p>
<p>
1. <b>Taxonomic and phylogenetic stewardship</b> of global mammals. This work includes leadership of the <a href="https://www.mammaldiversity.org/">Mammal Diversity Database</a> in alliance with the American Society of Mammalogists for keeping track of species- and higher-level taxonomic changes for ~6,500 currently recognized species. It also includes improving upon the <a href="http://vertlife.org/data/mammals/">mammal tree of life</a>, including the development of more sophisticated tools for updating the species name labels for the millions of primary-source observations in global databases (e.g., GBIF, GenBank).
</p>
<tr><td><img src="https://n8upham.github.io/images/Fig1_NDexp_TopoCons_24Oct2018.jpg" width="1000px" style="padding-bottom: 20px; padding-top: 20px;"/></td></tr>
<br>
<p></p>
<p>
2. <b>Digitally connecting ecological data</b> from disparate publications, toward building a comprehensive knowledge graph for mammal host-pathogen interactions. This work aims to connect the underlying evidence in support of given ecological interactions with the interaction data itself. This evidence extends from the taxonomic (e.g., voucher specimens of the species in question) to the methodological (e.g., antibodies, gene fragments, or live pathogen recovered in the host animal), with large impact on the resulting confidence of modeling efforts using those data. We have published an initial perspective (<a href="https://doi.org/10.1016/S2542-5196(21)00196-0"><i>The Lancet Planetary Health</i> 2021</a>) and have an <a href="https://reporter.nih.gov/search/jHonNExiyEulTWBDs1zc-Q/project-details/10289637">NIH-funded R21</a> to develop this work further.
</p>
<tr><td><img src="https://n8upham.github.io/images/Fig1_policyForums_specific_v4_toPublish.png" width="1000px" style="padding-bottom: 20px; padding-top: 20px;"/></td></tr>
<br>
<p></p>
<p>
3. <b>Querying how species traits influence both gene flow and pathogen sharing</b>, focusing on clades of Nearctic and Neotropical rodents. Using data collected in the field (deserts of the US Southwest and Argentina), in museum collections (modern and fossil specimens), and in the lab (population genomics), this work seeks to understand the impact species dispersal ability (=vagility) on ecological interactions. We are studying the trait correlates of vagility (e.g., limb morphology, physiology, and home range allometry) to investigate their dual influences upon historical patterns of gene flow and modern networks of pathogen sharing. This work was inspired by initial tests on the mammal tree (<a href="https://doi.org/10.1101/504803">bioRxiv 2020</a>) and prior work studying the largest genome of any mammal, the <i>Tympanoctomys</i> vizcacha rats of Argentina (<a href="https://academic.oup.com/gbe/article/9/6/1711/3954029/Evolution-of-the-Largest-Mammalian-Genome?guestAccessKey=fbcc7c6e-bebb-431c-88eb-1523a734f60e"><i>Genome Biology and Evolution</i> 2017).

<tr><td><img src="https://n8upham.github.io/images/Fig2_batFig_policyForums_v9_clip-all2.png" width="1000px" style="padding-bottom: 20px; padding-top: 20px;"/></td></tr>




