---
layout: default
title: research
---

<article class="row post-content">

<p>
<h2>
<strong>Research framework</strong>
</h2>
<p>
The questions motivating my research program extend across scales from local populations to regional clades of species to ask how processes of gene flow and selection change through time to influence species birth, death, and adaptation. The spectacular gradients of biodiversity we observe in modern habitats—from the tropics to the poles, across elevations, and between diverse and depauperate lineages of organism—are the current products of spatial and temporal variances in the balance of speciation, extinction, and dispersal. 
</p><blockquote>
Yet those processes of diversification are neither well understood (i) in a descriptive context, in terms of rates and magnitudes of change across regions and clades; nor (ii) in a causal context, in terms of their underlying ecological and historical drivers. To what extent do evolutionary rate variations follow ecological and geographic rules across the tree of life? What are the relative roles of contingent events (e.g., long-distance dispersal, area of endemism) and environmental interactions (both biotic and abiotic) as drivers of species diversification and trait adaptation? I address these questions by integrating genomic, paleontological, and bioinformatic tools to study clades of mammals, notably rodents, as they are particularly well matched to studying ecological diversification and are directly relevant to human health and economic goals (e.g., as disease reservoirs and seed dispersers). 
</blockquote><p>
At the center of these efforts are timetrees, i.e., time-calibrated molecular phylogenies detailing evolutionary hypotheses for the branching relationships among species through time. Geological ages, especially from fossils, are essential for scaling relative rates of molecular substitution to time, a key step that allows disparate evolutionary processes to be compared under the common framework of temporal rates. This primacy of fossil data integration therefore suggests the need to target fossil-rich lineages for studying the tempo and mode of diversification, among which Mammalia stands out as exceptional. Diverse radiations of rodents have been focal for several components of my research program, which are detailed below.
</p>

<h2>
<strong>Rodent radiations: ecology, diversification, and biogeography</strong>
</h2>
<h3><i>In the tropical Americas – [regional- to continental-scale; major rodent clades]
</h3></i>
<p>
Projects focusing on the rodent clade Caviomorpha (guinea pigs, spiny rats, and their relatives; ~250 living species) are the center of my ongoing studies of evolutionary rates and their underlying drivers. As the first rodent lineage to reach South America, caviomorphs are a fascinating example of a continental radiation that took place without close coexisting relatives. To what extent did that ecological opportunity influence their tempo and mode of trait adaptation and species diversification? The well-studied Cenozoic fossil record of caviomorphs is a key asset is this research, and was a primary draw for me to begin these studies.
</p><p style="text-align:justify">
<img src="http://n8upham.github.io/images/research_FigCavios.jpg" width="150px" style="float:right" />
Current work focuses on testing for signatures of adaptively radiating clades in Caviomorpha, mapping upon the timetree their exceptional diversity of body sizes (60 g to 60 kg, including capybaras, the world’s largest rodents) and ecological life modes (e.g., tree-dwelling to burrowing and semi-aquatic forms). I am using phylogenetic comparative methods to model rates of species diversification and eco-morphological disparification, assessing changes in ecospace occupation through time (Figure; ms in revision). The need to rigorously define criteria and test expectations for adaptively radiating clades has thus far been a key insight from this work, spurring the development of new methods.
</p><p>
Key collaborators: B. Patterson, B. Evans, D. Verzi, M. Perez, R. Norris
</p>


<h3><i>In North American deserts – [regional-scale; heteromyid and cricetid rodents]
</h3></i>
<p>
My early work on projects in the Great Basin Desert focused on questions of population genetics, biogeography, and behavioral ecology that I have since built upon. That early fieldwork in search of elusive kangaroo mice (<i>Microdipodops</i>) revealed the striking diversity of nocturnal rodents inhabiting sagebrush steppe, and demonstrated to me key concepts of niche partitioning, habitat specialization, and how species assemble in rank-abundance hierarchies. In lab, our phylogeographic work revealed five genetic units of at least 3-million years divergence within the then-two recognized species of kangaroo mice. Historical gene flow from south-to-north suggested range expansion from glacial refugia, but the pre-Pleistocene divergences among phylogeographic units implied older barriers to gene flow. In a related project that leveraged our rodent activity data from seven years of Great Basin trapping, I investigated the influence of moon brightness and phase on trap success of nocturnal species to test the hypothesis of moonlight avoidance.
</p><p>
Key collaborators: J. Hafner, J. Light, E. Reddington, C. Torres
</p>

<h2>
<strong>Evolution of genome expansion in vizcacha rats of Argentina</strong>
</h2>
<h3><i>In the Monte Desert and Patagonia – [local-scale; subclade of octodontid rodents]
</h3></i>
<p>
Our main goal for this ongoing project is to test, for the first time using genomic data, hypotheses regarding the origin of their inflated genomes. Specifically, whether their large size is the product of whole genome duplication (WGD), as previously reported, or else stems from the accumulation of repetitive genomic elements (RGE). During field expeditions in 2014 and 2015, we collected fresh tissues from Tympanoctomys (4n?=102) and their sister taxon Octomys (2n=56) for RNA sequencing. That fieldwork was the topic of a [popular science article I wrote](http://www.thehumangeographic.com/vizcacha-rat/). We are presently assembling and analyzing these transcriptomes, and are planning additional fieldwork and genome sequencing pending our funding application to the National Geographic’s Committee for Research and Exploration.
</p><p>
Key collaborators: B. Evans, B. Golding, A. Ojeda, R. Ojeda
</p>

<h2>
<strong>Biodiversity and extinction patterns in Caribbean endemic mammals</strong>
</h2>
<h3><i>In the Greater Antilles – [regional- to local-scale; bats, hutias, and solenodon]
</h3></i>
<p>
My goals for studying Caribbean mammal extinctions focus on understanding (i) the historical dynamics and causes of these events, including the relative roles of human activity (hunting, invasive species) and climatic changes; and (ii) the ecological consequences of native species loss and replacement by invasive species. Thus far, several projects have blossomed as a result of these efforts. In June 2015, I was the primary organizer for a [2-part featured symposium](http://conferences.k-state.edu/mammalogists/program/symposia-workshops/) at the American Society of Mammalogists meeting to synthesize ideas about “how we got here” and “what to do now” in the context of Caribbean mammal biodiversity. Eight invited speakers from five countries gave talks that will be published next year in a Special Feature section of the <i>Journal of Mammalogy</i>. We are planning additional group efforts, including an invited synthesis paper for the 2018 <i>Annual Reviews in Ecology, Evolution, & Systematics</i>. 
</p><p>
Key collaborators: R. Borroto, B. Lim, L. Davalos, S. Turvey, S. Cooke, P.-H. Fabre
</p>
<h2>
<strong>Phylogenetics and trait diversification of Mammalia</strong>
</h2>
<h3><i>Worldwide – [global-scale, all mammals]
</h3></i>
<p>
Now nearly 10 years old, the supertree-based analyses of Bininda-Emonds et al. (2007, plus subsequent taxonomic updates) is still the most complete species-level phylogeny of mammals (cited ~250 times since 2014). However, as the product of multiple phylogenies pasted together, the branch lengths in that supertree are inaccurate and thus render problematic any attempt to analyze evolutionary rates in global Mammalia. To obtain an improved estimate of tempo and mode in Mammalia, I am using a DNA supermatrix-based approach to maximize species-level sampling across 30 genes.
</p><p>
Key collaborators: W. Jetz, J. Esselstyn, G. Thomas, R. Guralnick
</p>

